core/spec/controllers/refinery/admin/dummy_controller_spec.rb

```diff
@@ -2,6 +2,14 @@
 
 module Refinery
   module Admin
+    class Engine < ::Rails::Engine
+      isolate_namespace Refinery::Admin
```

Comment:
This is insane :(

`isolate_namespace` is meant to be used in engine as we do [already](https://github.com/refinery/refinerycms/blob/1a8278498a5d9eaa7a0db4946d2055a9ee83b634/core/lib/refinery/core/engine.rb#L6).

I'm working on proper fix for this.